flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/config/MySqlSourceConfigFactory.java

```diff
@@ -318,6 +318,13 @@ public MySqlSourceConfig createConfig(int subtaskId) {
             jdbcProperties = new Properties();
         }
 
+        // use cursor-based streaming to retrieve a set number of rows
+        // https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-implementation-notes.html
+        if (fetchSize > 0) {
+            jdbcProperties.setProperty("useCursorFetch", "true");
+            props.setProperty(QUERY_FETCH_SIZE.name(), String.valueOf(fetchSize));
```

Comment:
The `QUERY_FETCH_SIZE` property is useless for mysql.

Reply:
Hi @GOODBOY008  I use the mysql 8.0.21 driver and it works